common/setup.py

```diff
@@ -4,7 +4,8 @@
 PYTHON_SRC = 'src/main/python'
 
 install_requires = [
-  "dvp-api == 1.7.0",
+    "dvp-api == 1.7.0",
+    "six >= 1.16, < 1.17",
```

Comment:
why not set this to an exact version?

Reply:
For all setup.py, we tried to keep a range for bug fixes to be automatically be pulled during `pip install` or `pip upgrade`. This will help fix any bug fixes related to any dependencies without the need for a new dvp version release.

Also, having a range will help developers to install dependencies from any Arti-factory (other than pypi and testpypi) based on restriction they have in organisation(where they use self-hosted Arti-factory for packages).

For now we have kept the minimum as latest version (as we knew it is already being used by developers based on previous >= which installed the latest dependency).
We have the maximum as next `minor` version.

In future the range can be widen by changing the maximum supported version based on testing (without changing minimum).